functions/utils/cache.test.ts

```diff
@@ -0,0 +1,42 @@
+import CacheMock from 'browser-cache-mock'
+
+import Cache from './cache'
+
+const cacheMock = new CacheMock()
+
+beforeAll(() => {
+  const globalAny: any = global
+  globalAny.caches = {
+    open: async () => cacheMock,
+    ...cacheMock,
+  }
+})
+
+test('Should put cache properly', async () => {
+  //wait for the server to start
```

Comment:
These tests should not be server-dependent. You can instantiate your own `Response` objects inline, without any need for a server.